emukit/examples/profet/meta_benchmarks/meta_fcnet.py

```diff
@@ -0,0 +1,96 @@
+import pickle
+import numpy as np
+
+try:
+    import torch
+except ImportError:
+    raise ImportError('pytorch is not installed. Please installed version it by running pip install torch torchvision')
+
+from functools import partial
+from typing import Tuple
+
+from emukit.core import ContinuousParameter, ParameterSpace
+from emukit.core.loop.user_function import UserFunctionWrapper
+from emukit.examples.profet.meta_benchmarks.architecture import get_default_architecture_classification, \
+    get_default_architecture_cost
+
+
+def meta_fcnet(fname_objective: str, fname_cost: str, noise: bool=True) -> Tuple[UserFunctionWrapper, ParameterSpace]:
+    """
+    Interface to the Meta-FCNet benchmark which imitates the hyperparameter optimization of a
+    fully connected neural network on OpenML like classification datasets.
+    Offline generated function samples can be download here:
+
+    http://www.ml4aad.org/wp-content/uploads/2019/05/profet_data.tar.gz
+
+    NOTE: make sure that the index for the objective function and the cost function match,
+    e.g for sample_objective_i.pkl and sample_cost_i.pkl the index i should be the same.
+    Each of those files contain the parameters of a neural network which represents a single function sample and
+    some additional information, such as the mean and variance of the input normalization.
+
+    For further information about Profet and the generated meta-surrogate benchmarks see:
+
+    Meta-Surrogate Benchmarking for Hyperparameter Optimization
+    A. Klein and Z. Dai and F. Hutter and N. Lawrence and J. Gonzalez
+    arXiv:1905.12982 [cs.LG] (2019)
+
+    :param fname_objective: filename for the objective function
+    :param fname_cost: filename for the cost function
+    :param noise: determines whether to add noise on the function value or not
+    :return: Tuple of user function object and parameter space
+    """
+    parameter_space = ParameterSpace([
+        ContinuousParameter('lr', 0, 1),  # original space [1e-6, 1e-1]
+        ContinuousParameter('batch_size', 0, 1),  # original space [8, 128]
+        ContinuousParameter('n_units_1', 0, 1),  # original space [16, 512]
+        ContinuousParameter('n_units_2', 0, 1),  # original space [16, 512]
+        ContinuousParameter('dropout_1', 0, 1),  # original space [0, 0.99]
+        ContinuousParameter('dropout_2', 0, 1),  # original space [0, 0.99]
+    ])
+    data = pickle.load(open(fname_objective, "rb"))
```

Comment:
Are these some kind of data values in this file? Maybe it would be better to have them in some kind of csv or json, instead of brittle pickle?